server/backend/quiz/views.py

```
@@ -0,0 +1,108 @@
+        category = get_object_or_404(Category, pk=category_id)
+        # Get 20 random questions for the category
+        questions = Question.objects.filter(category=category).order_by('?')[:20]
+        
+        return Response({
+            'category_id': category.id,
+            'category_name': category.name,
+            'questions': QuestionSerializer(questions, many=True).data
+        })
+
+class SubmitQuizAPIView(APIView):
+    permission_classes = [IsAuthenticated]
```

Ruff (0.8.2)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)